filesystem/squashfs/squashfs_internal_test.go

```diff
@@ -68,7 +68,7 @@ func TestValidateBlocksize(t *testing.T) {
 
 func TestParseXAttrsTable(t *testing.T) {
 	// parseXattrsTable(bUIDXattr, bIndex []byte, offset uint64, c compressor) (*xAttrTable, error) {
-	b, offset, err := testGetInodeMetabytes()
+	b, _, err := testGetInodeMetabytes()
```

Comment:
ahh this is why linting is failing, looks like this was the only place using the `offset`